### PR TITLE
chore: update MacOS executor to M1

### PR DESCRIPTION
**Pull request by bnpfeife**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -56,7 +56,9 @@ commands:
           condition:
             equal: [ darwin, << parameters.os >> ]
           steps:
-            - run: 'sh ./scripts/installgo_mac.sh'
+            - run: |
+               ./scripts/install-rosetta.sh
+               ./scripts/installgo_mac.sh
```
Comment (jeffreyssmith2nd) [inline .circleci/config.yml:61]: Is `installgo_mac.sh` intentionally empty? And if so, what purpose does it serve?

**Comment by powersj** [comment]
hey I do already have #13932 and working through a test issues which avoids using rosetta as I wonder if we are just kicking the need to update the binaries down the road.

**Comment by bnpfeife** [comment]
> hey I do already have #13932 and working through a test issues which avoids using rosetta as I wonder if we are just kicking the need to update the binaries down the road.

Oh, I missed that PR. I'll close this in favor of that one.

File: .circleci/config.yml (hunk 1 above)
Comment (bnpfeife) [reply .circleci/config.yml:61]: Oh, that's weird. It looks like it's a rendering issue on GitHub's end. The file is visible if you click on "View File".

**Pull request closed without merging**